software/verification/config_test.json

```diff
@@ -0,0 +1,22 @@
+{
+ "references": [
+   {
+     "model": "Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.SetPoints.Validation.ZoneTemperatures",
+     "generate_json": true,
+     "sequence": "TZonSet",
+     "pointNameMapping": "realControllerPointMapping.json",
+     "run_controller": false,
```

Comment:
I noted that sometimes camel case is used as in `pointNameMapping` and sometimes underscores are used as in `run_controller`. I see no reason for not being consistent. I suggest to change them all to camel case (we also use camel case in Modelica and CDL), but either is fine, as long as it is internally consistent.

Please also make this change in the OBC paper.